angel.py

```diff
@@ -75,131 +78,186 @@ def read_csv(filename):
     return person_list
 
 
-def convert_to_player(row):
+def generalized_convert_to_player(row, column_names_index_dict):
```

Comment:
why does the name contain "generalized"? is there still a relevant non-generalized version?